tests/modules/programs/neovim/runtime.nix

```diff
@@ -19,13 +19,20 @@ with lib;
           };
         }
       ];
+    } // lib.mkMerge [
+      {
+        extraPython3Packages = ps: with ps; [ jedi pynvim ];
+        extraLuaPackages = ps: with ps; [ luacheck ];
+      }
+      {
+        extraPython3Packages = with pkgs.python3Packages; [ jedi pynvim ];
+        extraLuaPackages = with pkgs.lua51Packages; [ luacheck ];
```

Comment:
what if you use a function here too ?

Reply:
I explicitly added both ways of using the option to make sure that as long as we accept them, they both work.
To avoid a regression as the one this PR is fixing where one of the documented types is not actually accepted.

The variant where we use a function is just above.